SU2_CFD/src/solver_direct_mean.cpp

```diff
@@ -13508,6 +13508,12 @@ void CEulerSolver::LoadRestart(CGeometry **geometry, CSolver ***solver, CConfig
   
   unsigned short skipVars = geometry[MESH_0]->GetnDim();
 
+  /*--- Store the number of variables for the turbulence model
+   (that could appear in the restart file before the grid velocities). ---*/
+  unsigned short turbVars = 0;
+  if (turb_model == SA || turb_model == SA_NEG) turbVars=1;
```

Comment:
Since there are multiple SA variants and only the SST model has 2 eqns., to be safe, I would recommend something like

if(turb){
if(sst) vars = 2;
else vars = 1;
}

Long term, I would like to remove all index jumping as we have here in favor of either a method that accesses the correct index automatically, or accessing it by string tag instead, which we have in the restart file.

Reply:
I updated the if statement accordingly. I also agree that we should change that soon by accessing it by strings ...